packages/common/src/visualizations/CartesianChartDataModel.ts

```diff
@@ -176,10 +189,14 @@ export class CartesianChartDataModel
                               )
                             : undefined,
                     },
-                    color: this.getSeriesColor(
-                        seriesColumn,
-                        possibleXAxisValues,
-                    ),
+                    color:
+                        (display?.series &&
+                            display.series[seriesColumn]?.color) ||
+                        CartesianChartDataModel.getDefaultColor(
+                            index,
+                            orgColors,
+                        ),
+                    // this.getSeriesColor( seriesColumn, possibleXAxisValues, orgColors),
```

Comment:
do we need this then? can remove the comment, right?

Reply:
I replace this method with a new one. But the old one has some specific logic for saving so I was not sure if we should remove it